### PR TITLE
Add Zinc and Brass

### LINKED ISSUE #1209: Add Zinc Ore/Brass Alloy
Turns out we are going to need Zinc per #1205, for batteries. I brainstormed a few other uses for Zinc as well, one of which includes Brass:

Planned/Potential Uses:
- [x] Batteries (#1205)
- [ ] Zinc-Chloride - Wait on Chem Lab - Late-Game replacement for Creosote (wood treatment only, not fuel)
- [ ] Uses in Boilers maybe?
- [x] Brass (see Below)

Planned/Potential Uses for Brass:
- [x] Replace Gold-Plate Gear
- [ ] Use for Hobbyist's Steam Engine (if Kept)
- [x] Alternate Recipe for Gear Bearings
- [ ] Machinery
- [ ] Pipe Fittings and Valves (Usually use a lead addition or advanced precision, but this is Minecraft, we can go a little simpler here)
- [x] Alternate Recipe for Sturdy Casings (Forestry Integration, possibly?)
- [x] Aesthetics - Round out the Steam Punk look with Brass Blocks, etc.

### PR DESCRIPTION
**Pull request by Generalcamo**
Adds Zinc and Brass to the game. They are currently of limited utility, but more uses will pop up as batteries are implemented, 10.3 is worked on, and we think of them.

Replaces Gold-Plated Gears, possibly breaking to other mods that use them (probably none).

Closes #1209

### CODE REVIEW

**Comment by liach** [comment]
See #1209

**Review by liach — APPROVED** [review]
Looks fine in the first glance but don't merge it.

**Comment by Generalcamo** [comment]
Brass and Zinc respectively

File: src/main/java/mods/railcraft/common/worldgen/GeneratorMineZinc.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+/**
+ * Created by GeneralCamo on 5/16/2017.
+ *
+ * @author GeneralCamo
+ *         Created for Railcraft <http://www.railcraft.info>
+ */
+public class GeneratorMineZinc extends GeneratorMine {
+
+    private static final int Y_LEVEL = 40;
```
Comment (CovertJaguar) [inline src/main/java/mods/railcraft/common/worldgen/GeneratorMineZinc.java]: I suggest moving this to 30 so it generates on the same level as Lead, they are commonly found together.

**Review by CovertJaguar — CHANGES_REQUESTED** [review]

**Review by CovertJaguar — APPROVED** [review]

**Comment by liach** [comment]
@Generalcamo You forgot to add localization

**Pull request merged**